public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Text Level Elements</title>
+    </head>
+    <body>
+        <p>
+        Web development is a broad term for the work involved in developing a web 
+        site for the <strong>Internet (World Wide Web) or an intranet (a private network).</strong>
+        Web development can range from developing the simplest static single page
+        of plain text to the most complex web-based internet applications, 
+        electronic businesses, and social network services. A more comprehensive
+        list of tasks to which web development commonly refers, may include <em>
+        web engineering, web design, web content development, client liaison, 
+        client-side/server-side scripting, web server and network security 
+        configuration, and e-commerce development.</em> Among web professionals,
+        "web development" usually refers to the main non-design aspects of
+        building web sites: writing markup and coding. Most recently Web
+        development has come to mean the creation of content management systems
+        or CMS. These CMS can be made from scratch, proprietary or open source.
+        In broad terms the CMS acts as middleware between the database and the 
+        user through the browser. A principle benefit of a CMS is that it allows
+        non-technical people to make changes to their web site without having 
+        technical knowledge.
+        </p>
+    </body>
+</html>
```

Comment:
Great job, Priscilla! I can tell that you are understanding the main concepts.
Don't forget to validate all your assignments from now on!